docs/usage/projects.md

```diff
@@ -44,6 +44,9 @@ The `.spec.namespace` field is optional and will be initialized if it's unset.
 The name of the resulting namespace will be generated and look like `garden-dev-5anj3`, i.e., it has a random suffix.
 It's also possible to adopt existing namespaces by labeling them `gardener.cloud/role=project` and `project.gardener.cloud/name=dev` beforehand (otherwise, they cannot be adopted). 
 
+When deleting a Project resource, the corresponding namespace is also deleted. 
+To keep a namespace after project deletion, the namespace can be labeled with `namespace.gardener.cloud/keep-after-project-deletion`.
```

Comment:
Can you state that this is only possible to be done by operators/administrators because end-users cannot directly update the `Namespace` corresponding to their `Project`?